universe/ValueRef.cpp

```diff
@@ -611,6 +611,11 @@ namespace ValueRef {
         } else if (property_name == "CurrentTurn") {
             return CurrentTurn();
 
+        } else if (property_name == "Attack") {
+            if (TemporaryPtr<const Fleet> fleet = boost::dynamic_pointer_cast<const Fleet>(object))
+                return fleet->Damage();
+            if (TemporaryPtr<const Ship> ship = boost::dynamic_pointer_cast<const Ship>(object))
+                return ship->TotalWeaponsDamage();
```

Comment:
As I noted in the commit message, for planets there are already the Defense and MaxDefense meter properties available.  In some ways it might be convenient to have this cover planets as well, but then which meter would you use, Defense or MaxDefense?  It seems to me best to leave planets out of this; someone scripting for them should explicitly choose which meter they are interested in (or both).